src/components/Timeline/Types.ts

```diff
@@ -166,6 +168,11 @@ export interface TimelineControlsProps {
   onVolumeChange: TimelineProps["onVolumeChange"];
 }
 
+export interface TimelineCustomControls {
+  position: "left" | "right" | "left-center" | "right-center";
```

Comment:
do `"left" | "right" | "left-center" | "right-center"` map to anything? or get set from something else? would it be possible to change left-center and right-center to leftCenter and rightCenter respectively? this would make it so we can reference them above as `customControls?.rightCenter` instead of `customControls?.['right-center']`

Reply:
fair point, thank you